cmd/loop/quote.go

```diff
@@ -31,11 +32,22 @@ var quoteCommand = cli.Command{
 				"setting this flag might result in a lower " +
 				"swap fee.",
 		},
+		cli.BoolFlag{
+			Name: "in",
+			Usage: "Indicate you want to get the quote for a " +
+				"loop in instead of loop out.",
+		},
```

Comment:
Should it be like this, or is it better (long term) to convert to something like `loop in --quote`?

Reply:
I added the subcommands `in` and `out` to the `quote` command. Now it's quite similar to how the swaps themselves work.